### DISCUSSION TITLE
AppImage in Ubuntu 24.04

### DISCUSSION THREAD

**Discussion by Muffinxm347** [OrcaSlicer/OrcaSlicer#6318] [Dev]
Upgraded to Ubuntu 24.04 and the app image no longer function.  Returns error: ./OrcaSlicer_Linux_V2.0.0.AppImage
/tmp/.mount_OrcaSloTAd8w/bin/orca-slicer: error while loading shared libraries: libwebkit2gtk-4.0.so.37: cannot open shared object file: No such file or directory
It is unknown if this is a libwebkit2gtk issue or a kernel issue.
Any thoughts?

**Comment by aricknoteric** [comment]
I'm getting the same error message when trying to run within Terminal. I have 2 versions of fuse installed (both 64bit versions) and I have tried using Gear Lever to open the file also.
